actions-sms.php

```diff
@@ -432,226 +417,221 @@ function delstandnote($number, $standName, $message)
 
 function standNote($number, $standName, $message)
 {
-
-    global $db;
     $userId = getUser($number);
 
-
-    $result=R::getAll("SELECT standId FROM stands where standName=:standName", [':standName' => $standName]);
-    if ($result->num_rows!=1) {
+    $result = R::getAll("SELECT standId FROM stands where standName=:standName", [':standName' => $standName]);
+    if (count($result) != 1) {
         sendSMS($number, _("Stand")." ".$standName._("does not exist").".");
         return;
     }
 
-    $row =$result->fetch_assoc();
-    $standId=$row["standId"];
-
-    $result=R::getAll("SELECT userName from users where number=:number", [':number' => $number]);
-    $row =$result->fetch_assoc();
-    $reportedBy=$row["userName"];
+    $row = $result[0];
+    $standId = $row["standId"];
 
+    $result = R::getAll("SELECT userName from users where number=:number", [':number' => $number]);
+    $row = $result[0];
+    $reportedBy = $row["userName"];
```

Comment:
Isn't this the same (user) as $user_id ? Where is the `getUser` defined ? Can the user be retrieved at once with id and name ?